root/static/scripts/common/MB/Control/Autocomplete.js

```diff
@@ -721,7 +721,9 @@ MB.Control.autocomplete_formatters = {
         _(item.labels)
             .groupBy(getLabelName)
             .each(function (releaseLabels, name) {
-                var catalogNumbers = _(releaseLabels).map(getCatalogNumber).compact().sort().value();
+                var catalogNumbers = _(releaseLabels).map(getCatalogNumber).compact()
+                .sort()
+                .value();
```

Comment:
I think there should be an Indentation

Reply:
https://github.com/airbnb/javascript/blob/eslint-config-airbnb-v15.1.0/README.md#whitespace--chains